Framework/LiveData/src/ISIS/DAE/isisds_command.cpp

```diff
@@ -290,8 +290,9 @@ static int isisds_recv_command_helper(SOCKET s, char **command, void **data,
     return -1;
   }
   *command = reinterpret_cast<char *>(malloc(sizeof(comm.command) + 1));
-  strncpy(*command, comm.command, sizeof(comm.command));
-  (*command)[sizeof(comm.command)] = '\0';
+  const auto commCommandSize = sizeof(comm.command);
```

Comment:
if you move this up a line you can replace `malloc(sizeof(comm.command)+1)` with `malloc(commCommandSize+1)`